bin/windows-toolset.ps1

```diff
@@ -163,11 +151,29 @@ if ($buildSource) {
 cd $toolsDir
 
 # Download openssl
-Write-Host "Downloading http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma"
-(New-Object net.webclient).DownloadFile("http://buildsources.delivery.puppetlabs.net/windows/openssl/${opensslPkg}.tar.lzma", "$toolsDir\${opensslPkg}.tar.lzma")
+Write-Host "Downloading https://s3.amazonaws.com/kylo-pl-bucket/${opensslPkg}.tar.lzma"
```

Comment:
makes sense, also seems like something we can automate once everything is finalized